src/hooks/useBudgetByCategory.ts

```diff
@@ -1,8 +1,56 @@
+import { useEffect, useState } from 'react'
+
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+import toSnakeCase from 'lodash/snakeCase'
+
 import { PROPOSAL_GRANT_CATEGORY_ALL } from '../entities/Proposal/types'
 
+import { Governance } from './../clients/Governance'
 import { ProposalGrantCategory } from './../entities/Grant/types'
 
+type Budget = {
+  percentage: number
+  currentAmount: number
+  totalBudget: number
```

Comment:
I'd rather we call this properties `allocated` and `total`.

Also, I think the `percentage` name could be confusing depending on the context. How about `allocatedPercentage` ?